
(node:1868928) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.39',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-25 00:31:28.721 - [Info] MITM Version: 2
2022-10-25 00:31:28.722 - [Info] Auto Access Enabled: true
2022-10-25 00:31:28.723 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 00:31:28.729 - [Info] SSH man-in-the-middle server for 10.0.3.39 listening on 10.0.3.1:1003
(node:1886502) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
2022-10-25 00:32:44.040 - [Debug] [Connection] Attacker connected: 222.134.240.92 | Client Identification: SSH-2.0-Go
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.252',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-25 00:32:44.103 - [Info] MITM Version: 2
2022-10-25 00:32:44.105 - [Info] Auto Access Enabled: true
2022-10-25 00:32:44.106 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 00:32:44.114 - [Debug] listen EADDRINUSE: address already in use 10.0.3.1:1003
2022-10-25 00:32:44.114 - [Error] Another MITM instance or another program is already listening on this port
{"level":"error","message":"Forever detected script exited with code: 1"}
2022-10-25 00:32:45.717 - [Debug] [Auth] Attacker 222.134.240.92 trying to authenticate with 'none'
2022-10-25 00:32:45.718 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 00:32:46.262 - [Debug] [Auth] Attacker 222.134.240.92 trying to authenticate with 'publickey'
2022-10-25 00:32:46.264 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 00:32:46.264 - [Debug] [Auth] Attacker: 222.134.240.92 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 00:32:46.807 - [Debug] [Connection] Attacker closed the connection
2022-10-25 00:32:47.080 - [Debug] [Connection] Attacker connected: 222.134.240.92 | Client Identification: SSH-2.0-Go
